### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Liandyrf** [Shabinder/SpotiFlyer#109] [Ideas]
Hello, i am cuban, i love this app, i want to help with the spanish translation, i dont habe programming knowledge, but i cant translate the app lang with some help.

**Comment by Bmswad1** [comment]
How about using weblate
As long as it's used i can translate it into Arabic in no time

**Reply by mondlicht-und-sterne** [reply]
Is there a way (and if so, where) to translate this app?

**Reply by Bmswad1** [reply]
I know 3 ways
The first is for the developer to but the strings to be translated in a website like weblate for anyone to translate
The second is to get an xml file containing the strings to be translated directly from the developer and translate it using a text editor
And the third is to submit a pull request and I know nothing about it as I never could use it because of the lack of a pc

**Reply by mondlicht-und-sterne** [reply]
Weblate would be cool, of course. Is there already a link or can you already find it?

**Reply by mondlicht-und-sterne** [reply]
But the programmer would have to create the project at weblate, you can't do anything as a user.

**Reply by Shabinder** [reply]
I am looking into weblate :eyes:  and will finalize and update contribution guidelines this weekend.

Its really gr8 so many people eager to contribute.
Stay Tuned :v:

**Comment by Shabinder** [comment]
@Liandyrf , @dmswd, @mondlicht-und-sterne  if you are still interested , here you go : https://github.com/Shabinder/SpotiFlyer/blob/main/CONTRIBUTING.md

**Comment by mondlicht-und-sterne** [comment]
i don't understand the programming language with the fork and pull request. can someone send me the english translation (original language) ?

**Reply by Shabinder** [reply]
https://github.com/Shabinder/SpotiFlyer/blob/main/translations/Strings_en.properties

**Comment by mondlicht-und-sterne** [comment]
thank you
